addon/utils/bugsnag.js

```diff
@@ -0,0 +1,7 @@
+export function getMetaData(/* error, container */) {
+  return {};
+}
+
+export function getUser(/* container */) {
```

Comment:
Sounds good to me. I have some ideas of how it could be cleaner in the future but let's run with this for now.